pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -75,6 +79,7 @@ _multioutConfig() {
         --localedir=${!outputLib}/share/locale \
         $configureFlags"
 
+    echo $configureFlags
```

Comment:
Is this a leftover from debugging?

Reply:
good catch!

Reply:
I'm so happy it was the last commit and I don't have to do crazyness to edit the commit.
Thank you for spotting this!

Reply:
rebase fixup isn't craziness https://git-rebase.io/#fixup

Reply:
I didn't know about that, thanks! ❤️